tests/UKHO.S100PermitService.API.UnitTests/Controller/PermitControllerTests.cs

```diff
@@ -6,6 +6,7 @@
 using NUnit.Framework;
 using UKHO.S100PermitService.API.Controllers;
 using UKHO.S100PermitService.Common.Enum;
+using UKHO.S100PermitService.Common.Utilities;
```

Comment:
Can we make the namespace Events please?

Reply:
Sure, Changed to events